runtime/lua/vim/fs.lua

```diff
@@ -112,7 +112,11 @@ end
 ---@param ... string
```

Comment:
the docstring (line 108) mentions "with normalization"; let's edit that to explicitly mention: `slashes are normalized (redundant slashes are removed, and on Windows backslashes are replaced with forward-slashes)`

Reply:
**Suggestion**
```
--- Concatenate directories and/or file paths into a single path with normalization (redundant slashes are removed)
--- Windows: backward slashes are replaced with forward slashes
--- (e.g., `"foo/"` and `"/bar"` get joined to `"foo/bar"`)
--- (windows: e.g `"a\foo\"` and `"\bar"` are joined to `"a/foo/bar"`)
```

Reply:
would prefer the wording i gave

Reply:
Done